certification/pyxis/types.go

```diff
@@ -1,16 +1,16 @@
 package pyxis
 
 type CertImage struct {
-	ID                     string       `json:"_id,omitempty"`
-	Certified              bool         `json:"certified" default:"false"`
-	Deleted                bool         `json:"deleted" default:"false"`
-	DockerImageDigest      string       `json:"docker_image_digest,omitempty"`
-	DockerImageID          string       `json:"docker_image_id,omitempty"`
-	ImageID                string       `json:"image_id,omitempty"`
-	ISVPID                 string       `json:"isv_pid,omitempty"`
-	ParsedData             *ParsedData  `json:"parsed_data,omitempty"`
-	Architecture           string       `json:"architecture" default:"amd64"`
-	RawConfig              string       `json:"raw_config,omitempty"`
+	ID                string      `json:"_id,omitempty"`
+	Certified         bool        `json:"certified" default:"false"`
+	Deleted           bool        `json:"deleted" default:"false"`
+	DockerImageDigest string      `json:"docker_image_digest,omitempty"`
+	DockerImageID     string      `json:"docker_image_id,omitempty"`
+	ImageID           string      `json:"image_id,omitempty"`
+	ISVPID            string      `json:"isv_pid,omitempty"` // required
+	ParsedData        *ParsedData `json:"parsed_data,omitempty"`
+	Architecture      string      `json:"architecture" default:"amd64"`
+	// RawConfig            string       `json:"raw_config,omitempty"` // TODO: pyxis isn't ready for this API value yet.
```

Comment:
If you move the comment to the end, it will keep the formatting and minimize the changes above.

Reply:
They might have this fixed on Monday so I'll just put this PR onhold.

Reply:
I guess it makes more sense for people to comment this out locally if they are testing. Removed `/hold`